Library/Formula/osquery.rb

```diff
@@ -16,12 +16,11 @@ class Osquery < Formula
   depends_on :macos => :mavericks
 
   depends_on "cmake" => :build
-
-  depends_on "boost"
-  depends_on "gflags"
-  depends_on "openssl"
-  depends_on "rocksdb"
-  depends_on "thrift"
+  depends_on "boost" => :build
```

Comment:
Are these not used at runtime anymore? `:build` means that they are only needed while building.